tox.ini

```diff
@@ -45,5 +45,8 @@ exclude =
     RanDepict/__init__.py,
     Tests/test_functions.py,
 
-
+[pytest]
```

Comment:
I would rather not filter the warnings here, especially when they don't make the linter fail. 
Instead, I would prefer fixing the underlying problems or catch the warnings specifically with `warnings.catch_warnings` if we can't fix them for some reason. But this does not need to be done now

Reply:
Ok then lets remove that from tox.